src/Microsoft.Health.Fhir.Tests.Common/EmbeddedResourceManager.cs

```diff
@@ -13,7 +13,12 @@ public static class EmbeddedResourceManager
     {
         public static string GetStringContent(string embeddedResourceSubNamespace, string fileName, string extension)
         {
-            string resourceName = $"{typeof(EmbeddedResourceManager).Namespace}.{embeddedResourceSubNamespace}.{ModelInfoProvider.Version}.{fileName}.{extension}";
+            return GetStringContent(embeddedResourceSubNamespace, fileName, extension, ModelInfoProvider.Version.ToString());
```

Comment:
Since now this class has a dependency of ModelInfoProvider, I think we should use IoC container to inject this instance and remove the depdencies on the static instance (which I think is the direction we want to go eventually so might as well start cleaning it.)

Reply:
Agree, I'd accept this in a followup PR also though. I'll make a task for cleaning up the static ModelInfoProvider on a broader level

Reply:
Oh wait. Nevermind. This is for test.